src/app.js

```diff
@@ -1 +1,36 @@
+/* eslint-disable no-console */
 // write code here
+const fs = require('fs/promises');
+const path = require('path');
+
+async function moveFile() {
+  const [source, destination] = process.argv.slice(2);
+
+  if (!source || !destination) {
+    console.log(!source);
+    console.error('Please enter source file and destination');
+
+    return;
+  }
+
+  if (source === destination) {
+    return;
```

Comment:
It would be helpful to provide an error message here to inform the user that the source and destination are the same, which is why the operation is not proceeding.